FlashFeetAPI/src/main/java/org/example/trueApplication.java

```diff
@@ -30,8 +34,14 @@ protected SwaggerBundleConfiguration getSwaggerBundleConfiguration(trueConfigura
 
     @Override
     public void run(final trueConfiguration configuration,
-                    final Environment environment) {
-            environment.jersey().register(new JobRoleController());
+                    final Environment environment) throws SQLException {
+        environment.jersey().register(new JobRoleController());
+        try {
+            environment.jersey().register(new AuthController());
+        } catch (DatabaseConnectionException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
this shouldn't be needed, the AuthController should catch the DatabaseConnectionException rather than throwing it